{
  "comments": [
    {
      "key": {
        "uuid": "2db8c54a_29663881",
        "filename": "ci-scripts/dlrnapi_promoter/dlrn-promoter.sh",
        "patchSetId": 14
      },
      "lineNbr": 25,
      "author": {
        "id": 192
      },
      "writtenOn": "2019-09-13T07:01:11Z",
      "side": 1,
      "message": "we aren\u0027t using these in the https://review.rdoproject.org/r/#/c/21872/14/ci-scripts/infra-setup/roles/promoter/tasks/promotion_run.yml\n\nso lets remove if they aren\u0027t needed for this change",
      "range": {
        "startLine": 19,
        "startChar": 4,
        "endLine": 25,
        "endChar": 10
      },
      "revId": "ba322bb47b91796c41ac4b175855a15d6e209aba",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "d57ca129_17655503",
        "filename": "ci-scripts/infra-setup/roles/promoter/molecule/default/functional_tests.yml",
        "patchSetId": 14
      },
      "lineNbr": 122,
      "author": {
        "id": 192
      },
      "writtenOn": "2019-09-13T07:01:11Z",
      "side": 1,
      "message": "unnecessary?",
      "revId": "ba322bb47b91796c41ac4b175855a15d6e209aba",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "ffd628ca_c0efcc87",
        "filename": "ci-scripts/infra-setup/roles/promoter/tasks/promotion_run.yml",
        "patchSetId": 14
      },
      "lineNbr": 8,
      "author": {
        "id": 502
      },
      "writtenOn": "2019-09-12T21:36:44Z",
      "side": 1,
      "message": "(promoter_venv) [centos@centos7 ~]$ tail dlrnapi.log \nmounting dlrn.api:app on /\nImportError: No module named dlrn.api\nunable to load app 0 (mountpoint\u003d\u0027/\u0027) (callable not found or import error)\n*** no app loaded. going in full dynamic mode ***\n*** uWSGI is running in multiple interpreter mode ***\nspawned uWSGI worker 1 (and the only) (pid: 4802, cores: 1)\n--- no python application found, check your startup logs for errors ---\n[pid: 4802|app: -1|req: -1/1] 127.0.0.1 () {28 vars in 375 bytes} [Thu Sep 12 21:12:07 2019] GET /api/promotions \u003d\u003e generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)\n--- no python application found, check your startup logs for errors ---\n[pid: 4802|app: -1|req: -1/2] 127.0.0.1 () {28 vars in 375 bytes} [Thu Sep 12 21:34:45 2019] GET /api/promotions \u003d\u003e generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)",
      "range": {
        "startLine": 8,
        "startChar": 6,
        "endLine": 8,
        "endChar": 84
      },
      "revId": "ba322bb47b91796c41ac4b175855a15d6e209aba",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7",
      "unresolved": false
    }
  ]
}